app/usecases/password.py

```diff
@@ -11,8 +9,7 @@ async def verify_password(plain_password: str, hashed_password: str) -> bool:
     if hashed_password in CACHE:
         return CACHE[hashed_password] == plain_password
 
-    result = await asyncio.to_thread(
-        bcrypt.checkpw,
+    result = bcrypt.checkpw(
```

Comment:
from the [asyncio docs](https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task)
> Note Due to the [GIL](https://docs.python.org/3/glossary.html#term-GIL), asyncio.to_thread() can typically only be used to make IO-bound functions non-blocking. However, for extension modules that release the GIL or alternative Python implementations that don’t have one, asyncio.to_thread() can also be used for CPU-bound functions.

Reply:
as a result spinning up the thread and doing the event loop work takes additional cpu time; higher overall cost